.circleci/config.yml

```
@@ -65,6 +65,12 @@ commands:
               working_directory: test/acceptance/tests
               no_output_timeout: 2h
               command: |
+                # Enterprise tests can't run on fork PRs because they require
+                # a secret.
+                if [ -z "$CIRCLE_PR_NUMBER"]; then
```

```suggestion
                if [ -z "$CIRCLE_PR_NUMBER" ]; then
```
I tested it in #1018, and got `/bin/bash: line 2: [: missing `]'` [here](https://app.circleci.com/pipelines/github/hashicorp/consul-helm/3329/workflows/1a62922e-3eb2-4dcc-b92f-9c273e38534a/jobs/14607)

---

At least one of us tested this :D